protocol/x/vest/types/genesis.go

```diff
@@ -6,17 +6,29 @@ import (
 	rewardstypes "github.com/dydxprotocol/v4-chain/protocol/x/rewards/types"
 )
 
+var (
+	DefaultVestingStartTime = time.Date(2023, 1, 1, 0, 0, 0, 0, time.UTC).In(time.UTC)
+	DefaultVestingEndTime   = time.Date(2025, 1, 1, 0, 0, 0, 0, time.UTC).In(time.UTC)
+)
+
 // DefaultGenesis returns the default genesis state
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
 		// TODO(CORE-530): in genesis.sh, overwrite start and end times dynamically for testnets.
 		VestEntries: []VestEntry{
+			{
+				VesterAccount:   CommunityVesterAccountName,
+				TreasuryAccount: CommunityTreasuryAccountName,
+				Denom:           "dv4tnt",
```

Comment:
same here, don't we define some constnat?

Reply:
See above